packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -400,6 +408,22 @@ export const scheduleMicrotask: any =
           .catch(handleErrorInNextTick)
     : scheduleTimeout; // TODO: Determine the best fallback here.
 
+// -------------------
+//     requestAnimationFrame
+// -------------------
+export const scheduleAnimationFrame: any = localRequestAnimationFrame;
+export const cancelAnimationFrame: any = localCancelAnimationFrame;
+export const shouldScheduleAnimationFrame =
+  scheduleAnimationFrame != null && cancelAnimationFrame != null
+    ? function() {
+        return (
+          typeof window !== 'undefined' && typeof window.event === 'undefined'
```

Comment:
We already do this check here:

https://github.com/facebook/react/blob/aec575914a0fd76c9db8998ea62c3ee975de70d7/packages/react-dom/src/events/ReactDOMEventListener.js#L410-L513

I think the way I would do this is to add another type to the EventPriority enum, instead of checking `window.event` twice.